metrics/cli/docs/diagrams/deploy_command_sequence_diagram.puml

```diff
@@ -39,19 +44,27 @@ FirebaseService --> Deployer -- : sign in finished
 Deployer -> FirebaseService ++ : acceptTermsOfService()
 FirebaseService --> Bob : asks to accept terms of Firebase service
 Bob -> FirebaseService : accepts terms
-FirebaseService -> Deployer  -- : terms accepted
+FirebaseService --> Deployer  -- : terms accepted
 
 Deployer -> GCloudService ++ : createProject()
-
 GCloudService --> Bob : asks to enter the regionId
 Bob -> GCloudService : enters the regionId
-
 GCloudService --> Deployer -- : gcloud project has been created
 
-Deployer -> FirebaseService ++ : createWebApp(projectId)
+Deployer -> GCloudService ++ : configureProjectOrganization(projectId)
+GCloudService --> Bob : asks to configure the organization for the project
+Bob -> GCloudService : configures the organization for the project
```

Comment:
Would it be great to mention that this step is optional and the deployment will proceed even if the user skips this step?